### ISSUE TITLE
connector.RefundPayment - Deserialization error

### ISSUE THREAD

**Issue by PribylJaroslav** [gopaycommunity/gopay-dotnet-api#20]
// create connection   
GPConnector connector = new GPConnector("https://gw.sandbox.gopay.com/api", "1640607199", "eKS8ryh8");
connector.GetAppToken();

// refund payment - HERE COMES AN ERROR
var result = connector.RefundPayment(3166251348, 10);

Exception:
"Exception of type 'GoPay.GPClientException' was thrown."
"   at GoPay.GPConnector.Deserialize[T](String Content) 
     at GoPay.GPConnector.ProcessResponse[T](IRestResponse response)
     at  [MY_CODEFILE]"

**Comment by riadasllani** [comment]
We are having the same problem, currently we are using v1.1.17 of .NET SDK, that was the latest version for 6 months now.

The error we are having is
```
{
  \"Success\": false,
  \"Exception\": {
    \"Error\": {
      \"date_issued\": \"2022-08-16\",
      \"errors\": [
        {
          \"scope\": \"F\",
          \"field\": null,
          \"error_code\": 500,
          \"error_name\": null,
          \"message\": null,
          \"description\": \"RESTEASY003065: Cannot consume content type\"
        }
      ]
    },
    \"StackTrace\": \"   at GoPay.GPConnector.Deserialize[T](String Content)\r\n   at GoPay.GPConnector.ProcessResponse[T](IRestResponse response)\r\n   at GoPay.GPConnector.<>c__DisplayClass21_0.<RefundPaymentAsync>b__0()\r\n   at System.Threading.Tasks.Task`1.InnerInvoke()\r\n   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at [CLIENT_CODE]",
    \"Message\": \"Exception of type 'GoPay.GPClientException' was thrown.\",
    \"Data\": {},
    \"InnerException\": null,
    \"HelpLink\": null,
    \"Source\": \"GoPay.net-sdk\",
    \"HResult\": -2146233088
  }
}
```

Will updating version of .NET SDK to v1.1.18 help, or is this a general problem that is happening.

This is throwing even in our production application, and it was tested before and it was working.

Best regards!

**Comment by riadasllani** [comment]
We are having the same problem, currently we are using v1.1.17 of .NET SDK, that was the latest version for 6 months now.

The error we are having is

```
{
  \"Success\": false,
  \"Exception\": {
    \"Error\": {
      \"date_issued\": \"2022-08-16\",
      \"errors\": [
        {
          \"scope\": \"F\",
          \"field\": null,
          \"error_code\": 500,
          \"error_name\": null,
          \"message\": null,
          \"description\": \"RESTEASY003065: Cannot consume content type\"
        }
      ]
    },
    \"StackTrace\": \"   at GoPay.GPConnector.Deserialize[T](String Content)\r\n   at GoPay.GPConnector.ProcessResponse[T](IRestResponse response)\r\n   at GoPay.GPConnector.<>c__DisplayClass21_0.<RefundPaymentAsync>b__0()\r\n   at System.Threading.Tasks.Task`1.InnerInvoke()\r\n   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at [CLIENT_CODE]",
    \"Message\": \"Exception of type 'GoPay.GPClientException' was thrown.\",
    \"Data\": {},
    \"InnerException\": null,
    \"HelpLink\": null,
    \"Source\": \"GoPay.net-sdk\",
    \"HResult\": -2146233088
  }
}
```

Will updating version of .NET SDK to v1.1.18 help, or is this a general problem that is happening.

This is throwing even in our production application, and it was tested before and it was working.

**Comment by FSichinger** [comment]
@riadasllani does this issue happen after calling RefundPayment as well?

**Comment by FSichinger** [comment]
fixed version 1.1.19 has been pushed to nuget repository.
